plugins/intel-amt/README.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Plugin: Intel AMT
+---
+
+## Introduction
+
+This plugin is used to version number for the Intel AMT, typically CSME.
+
+If AMT is enabled and provisioned and the AMT version is between 6.0 and 11.2,
+and you have not upgraded your firmware, you are vulnerable to CVE-2017-5689 and
+you should disable AMT in your system firmware.
+
+This code is inspired by 'AMT status checker for Linux' by Matthew Garrett
+which can be found here: <https://github.com/mjg59/mei-amt-check>
```

Comment:
How about just making the `AMT status checker for Linux` a URL instead of `which can be found here: `

Reply:
Good idea, fixed.